sklearn/naive_bayes.py

```diff
@@ -115,6 +115,16 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    fit_prior : boolean
+        Whether to learn class prior probabilities or not.
+        If false, a uniform prior will be used.
+
+    class_prior : array-like, size=(n_classes,)
+        Prior probabilities of the classes. If specified the priors are not
+        adjusted according to the data.
```

Comment:
There is also something fishy with the multinomialNB test:
```
sklearn.tests.test_naive_bayes.test_mnnb ... /home/lemaitre/Documents/Work/Thesis/Code/toolbox/scikit-learn/sklearn/naive_bayes.py:459: RuntimeWarning: divide by zero encountered in log
  self.class_log_prior_ = (np.log(self.class_count_) -
ok
```
The problem should be that `class_count_` is initialize to zero and used at the first update in the log function. Since that this is a warning, the test is ok but that seems not right.